frontend/rolecall/src/app/performance/performance-editor.component.ts

```diff
@@ -532,28 +545,34 @@ export class PerformanceEditor implements OnInit, OnDestroy, AfterViewChecked {
       castAndPrimLength = this.segmentToCast.get(castUUID);
     }
     this.primaryGroupNum = castAndPrimLength[1];
-    this.castDnD ? this.castDnD.selectCast({uuid: castUUID, saveDeleteEnabled: false}) : '';
+    if (this.castDnD) {
+      this.castDnD.selectCast({uuid: castUUID, saveDeleteEnabled: false});
+    }
     this.updateGroupIndices(castAndPrimLength[0]);
     this.updateCastsForSegment();
     this.segmentLength = castAndPrimLength[2];
     this.changeDetectorRef.detectChanges();
-    this.castDnD ? this.castDnD.setBoldedCast(this.segmentToCast.get(castUUID)[1]) : '';
```

Comment:
What a strange syntax this was, wonder why they had it this way...

Reply:
I believe they were trying to do what your "better way" syntax below did.